website/docs/docs/cloud/secure/ip-restrictions.md

```diff
@@ -19,7 +19,7 @@ To configure IP restrictions, go to **Account Settings** → **IP Restrictions**
 - Deny IPs flagged by the Security team
 - Allow only VPN traffic but make an exception for contractors’ IP addresses
 
-IP restrictions will block all user requests done via the API (via personal user token) and the UI. Service tokens are exempt from IP restrictions and can still make requests to dbt Cloud API.
+IP restrictions will block all service tokens, user requests done via the API (via personal user token), and the UI.
```

Comment:
IP restrictions will block all service tokens requests, user requests done via the API (via personal user token), and the UI, if they are coming from blocked IP addresses.

Reply:
This is for you can add a note about integrations using serviceTokens. 
"For any versionControl system integrations inbound into dbt Cloud, ensure their IP addresses are added to the allowed list. Examples: Gitlab, ADO and Github"